README.md

```diff
@@ -1,17 +1,91 @@
-## Decentragit helper
-Implements a [git remote helper](https://git-scm.com/docs/git-remote-helpers) that stores repo information in a ChainTree.
+[![License](http://img.shields.io/:license-mit-blue.svg?style=flat-square)](http://badges.mit-license.org)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/quorumcontrol/dgit">
+    <img src="dgit-black.png" alt="Logo" width="150" height="125">
+  </a>
+
+  <h3 align="center">dgit</h3>
+
+  <p align="center">
+    <b>dgit</b> is an opensource project built by Quorum Control which combines
```

Comment:
I think "open-source" would be more correct here.

And should we link "Quorum Control" to our website?